app/graphql/mutations/album_search.rb

```diff
@@ -0,0 +1,13 @@
+require 'httparty'
```

Comment:
Instead of requiring the gem, it would be better to add it to the Gemfile, right?